examples/development/web2/index.js

```diff
@@ -1,12 +1,13 @@
 var CodeMirror = require("codemirror");
-require("./node_modules/codemirror/lib/codemirror.css");
+require("css!./node_modules/codemirror/lib/codemirror.css");
```

Comment:
You don't need to specify the loader here, you can do so in the config: https://github.com/phosphorjs/phosphor-tabs/blob/master/example/webpack.conf.js#L14